### ISSUE TITLE
[FEATURE] Create a todo from highlighted text

### ISSUE THREAD

**Issue by Akselmo** [pbek/QOwnNotes#2992]
#### Is your feature request related to a problem? Please describe.

I like to use tasks a lot to remind myself to do something. However no note taking app I know of has todo support like QOwnNotes does, which is what I really like! But making todo's is a bit cumbersome at the moment, since you have to hop through menus. I'd like to highlight some text and just turn that into a todo.

#### Describe the solution you'd like

The best solution for this IMO would be, if user has something like 

`- [ ] Do this task soon`

Then user highlights it, opens context menu (or presses a hotkey combo) and selects "Create todo item" and this would copy the text and create a todo item from it. It could also open the editor and when done, replace the highlighted text with a link to the todo. 

`- [ ] [Do this task soon](task://blablabl123123)`

Alternative solution, instead of opening the todo menu, could be somekind of "quick task" bar that appears, like with KMail. It could also have the reminder checkbox.

![image](https://github.com/pbek/QOwnNotes/assets/13801236/dffaabcf-9baf-458a-9a45-37faa1f01138)

#### Describe alternatives you've considered
The todo menu is ok, but I just wish I could access it easier, so I don't know what other alternative there is.

#### Additional context

Basically I hope for a more streamlined todo system that could be used in tandem with regular markdown faster than the current setup.

If this is scriptable, I would not mind at all making a script for this, but I could not find todo menu stuff being exposed in the scripting docs.

Thanks for the great app. :)

**Comment by pbek** [comment]
Nowadays, I only use Nextcloud Deck (with the QOwnNotes integration) for all that. The API is much better than CalDAV.

But the highlighted text will be used when you open the todo dialog in the "search or create" line-edit, then you just need to press "Enter" and a new item will be created. And you can also insert a link to that todo item in the note in that dialog...

The Nextcloud Deck integration works pretty much the same...

**Comment by Akselmo** [comment]
Oh, hah, I didn't realise. So the feature I want is already there. :joy: Thanks!

I'll check Nextcloud Deck as well! 

Closing this issue. :)

**Comment by pbek** [comment]
Great 😉
